view/template/acquisition/auto-verify.php

```diff
@@ -0,0 +1,70 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+    <title>Verify Your Identity</title>
+    <link rel="icon" href="images/favicon.ico">
+    <script src='https://www.google.com/recaptcha/api.js?' async defer></script>
+    <script type="text/javascript">
+        const ENDPOINT = 'https://api.lbry.io/user_email/confirm_v2'
+
+        var verifyUser = function(email, verification_token, recaptcha) {
+            const url = `${ENDPOINT}?email=${email}&verification_token=${verification_token+"x"}&recaptcha=${recaptcha}`;
+            fetch(url)
+                .then((response) => {
+                    if (response.error) {
+                        throw Error(response.error)
+                    }
+                    document.getElementById("title").textContent = "Done!"
+                    document.getElementById("verify").style.display = "none";
+                    document.getElementById("verify-success").style.display = "block";
+                })
+                .catch(error => {
+                    document.getElementById("verify").style.display = "none";
+                    document.getElementById("verify-error").style.display = "block";
+                    document.getElementById("verify-error-text").textContent = error.message;
+                })
+        }
+
+        var verifyCallback = function(response) {
+            const urlParams = new URLSearchParams(window.location.search);
+            const email = urlParams.get('email')
+            const verification_token = urlParams.get('verification_token');
+            // In the future we will have an `origin` query, to smartly redirect (or tell users they are verified)
+            // eg. if origin == "android" and device == "android" open the app, else say "your android app is verified"
+            verifyUser(email, verification_token, response);
+
+            document.getElementById("captcha-block").style.display = "none";
+            document.getElementById("verify").style.display = "block";
+        }
+
+        var expiredCallback = function(error) {
+            console.log("expired", error)
+        }
+    </script>
+</head>
+<body>
+<div style="display: flex; align-items: center; flex-direction: column;" class="text-center">
+    <img src="/img/lbry-dark-1600x528.png" style="max-height: 80px; margin-top: 50px; margin-bottom: 38px" alt="LBRY"/>
+    <h1 id="title">Almost Done!</h1>
+    <div id="captcha-block">
+        <p>Click the captcha to continue...</p>
+        <br/>
+        <div class="g-recaptcha" data-sitekey="6LcG_z0UAAAAAKBPDBhiJU_jI9cRNRiJwcUHq95u" data-callback="verifyCallback" data-expired-callback="expiredCallback"></div>
+    </div>
+
+    <div style="display: none; margin-top: 10px;" id="verify">
+        <p>Verifying...</p>
+    </div>
+    <div style="display: none; margin-top: 10px;" id="verify-error">
+        <p>There was an error</p>
+        <code id="verify-error-text"></code>
```

Comment:
We should probably add other text here if an error occurs, I'm not sure what though.

Reply:
1) Is there information from server that would be helpful to be displayed?
2) You can always encourage them to contact help@lbry.io

Reply:
Added a comment about contacting help@lbry.io if an error keeps happening.